charts/kargo/crds/kargo.akuity.io_projects.yaml

```diff
@@ -71,8 +71,6 @@ spec:
                       minLength: 1
                       pattern: ^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$
                       type: string
-                  required:
-                  - stage
```

Comment:
It doesn't look right that this got removed.

Same comment applies in other places.